.github/workflows/post-pr-comment.yml

```diff
@@ -1,12 +1,17 @@
-name: Post pull-request comment
+# This workflow is the second part of the process described in
+# .github/workflows/qhelp-pr-preview.yml
+# See that file for more info.
+
+name: Post PR comment
 on:
   workflow_run:
-    workflows: ["Query help preview"]
+    workflows: [Render QHelp changes]
     types:
       - completed
 
 permissions:
   pull-requests: write
+  actions: read
```

Comment:
The permission looks reasonable, I just wonder why it is needed, we're making the same API call as before, right? Or is this just to be a bit more explicit( which I think is a good thing).?

Reply:
In my testing, I needed this in order for `gh run download` to not throw `HTTP 403: Resource not accessible by integration`. This may be due to a configuration difference between my test repo and `github/codeql`, though. I guess it's good to be explicit regardless.